src/constants/routes.ts

```diff
@@ -1,20 +1,20 @@
 export enum Route {
   HOME = "/",
   REGISTER = "/register",
-  NOT_FOUND = "/*",
   ADMIN = "/admin",
   CART = "/cart",
   CATEGORY = "/category/:id",
+  LOGIN = "/login",
+  PRODUCTS_LIST = "/products",
+  PRODUCT = "/product/:id",
+  PRODUCTS_LIST_SEARCHED = "/products?search=",
```

Comment:
acho que fica mais legivel assim
```suggestion
  PRODUCT = "/product/:id",
  PRODUCTS_LIST = "/products",
  PRODUCTS_LIST_SEARCHED = "/products?search=",
```